openstates/de/bills.py

```diff
@@ -110,26 +132,28 @@ def scrape_legislators(self, session):
             'group': '',
             'filter': '',
         }
-        response = self.post(url=search_form_url, data=form, allow_redirects=True)
-        page = json.loads(response.content)
-        #print json.dumps(page, indent=4)
+
+        # response = self.post(url=search_form_url, data=form, allow_redirects=True)
+        page = requests.post(url=search_form_url, data=form, allow_redirects=True).json()
         if int(page['Total']) > 0:
             for row in page['Data']:
-                self.legislators[ str(row['PersonId']) ] = row
-                self.legislators_by_short[ str(row['ShortName']) ] = row
+                self.legislators[str(row['PersonId'])] = row
```

Comment:
Well, I was caching the legislators, we'll need them for sponsors and votes. `row['PersonId'])` is of type `int` and while scraping the page we don't get `int` type data. that's why I choose to cache all legislators data into `str` format. we can do this either way.